website/content/en/preview/concepts/nodeclaim.md

```diff
@@ -0,0 +1,138 @@
+6. Checks that the Node exists, and prepares it for use.  This includes:
+    * Waiting for the instance to return a provider ID, instance type, zone, capacity type,
+      and an allocatable status, indicating that the instance is ready.
+    * Checking to see if the node has been synced, adding a finalizer to the node (this provides the same
+      termination guarantees that all Karpenter nodes have), passing in labels, and updating the node owner references.
+    * Checking if the node has been synced, adding a finalizer to the node (providing Karpenter termination guarantees), passing in labels, and updating the node owner references.
+    * Making sure that the Node is ready to use. This includes such things as seeing if resources are registered and start-up taints are removed.
+    * Checking the nodes for liveliness.
+7. Registers the instance as a node in the Kubernetes cluster (registered). Example of log message at this stage:
+    ```
+    {"level":"INFO","time":"2024-06-22T02:24:39.998Z","logger":"controller","message":
+       "registered nodeclaim","commit":"490ef94","controller":"nodeclaim.lifecycle",
+       "controllerGroup":"karpenter.sh","controllerKind":"NodeClaim","NodeClaim":
+       {"name":"default-sfpsl"},"namespace":"","name":"default-sfpsl","reconcileID":
+       "4ae2c003-883c-4655-98b9-45871223a6a0",
+       "provider-id":"aws:///us-west-2b/i-08a3bf1cadb205c7e",
+       "Node":{"name":"ip-192-168-170-220.us-west-2.compute.internal"}}
+    ```
+    Finally, you can see the Node is now ready for use.
+    ```
+    {"level":"INFO","time":"2024-06-22T02:24:52.642Z","logger":"controller","message":
+       "initialized nodeclaim","commit":"490ef94","controller":
+       "nodeclaim.lifecycle","controllerGroup":"karpenter.sh","controllerKind":
+       "NodeClaim","NodeClaim":{"name":"default-sfpsl"},"namespace":
+       "","name":"default-sfpsl","reconcileID":
+       "7e7a671d-887f-428d-bd79-ddf603290f0a",
+       "provider-id":"aws:///us-west-2b/i-08a3bf1cadb205c7e",
+       "Node":{"name":"ip-192-168-170-220.us-west-2.compute.internal"},
+       "allocatable":{"cpu":"7910m","ephemeral-storage":"18242267924",
+       "hugepages-2Mi":"0","memory":"14320468Ki","pods":"58"}}
+    ```
+    At this point, the node is considered ready to go.
+
+If a node doesn’t appear as registered after 15 minutes since it was first created, the NodeClaim is deleted.
+Karpenter assumes there is a problem that isn’t going to be fixed.
+No pods will be deployed there. After the node is deleted, Karpenter will try to create a new NodeClaim.
+
+## NodeClaim drift and disruption
+
+Although NodeClaims play a role in replacing nodes that drift or have been disrupted,
+as someone using Karpenter, there is nothing particular you need to do with NodeClaims
+in relation to those activities.
+Just know that, if a Node become unusable and a new NodeClaim needs to be created,
+it follows the same node creation process just decribed.
```

Comment:
```suggestion
Although NodeClaims play a role in replacing nodes that drift or have been disrupted,
as a Karpenter user, NodeClaims should not be modified, and should only be a tool for monitoring.
```

Reply:
Should not be modified, or are immutable?

Reply:
Done